app/models/racer.rb

```diff
@@ -1,21 +1,20 @@
-require 'uri'
-
 class Racer < ActiveRecord::Base
   has_many :race_entries
-  
+
+  VALID_EMAIL_REGEX = /\A[\w+\-.]+@[a-z\d\-.]+\.[a-z]+\z/i
+
   before_save {
     self.email = self.email.downcase  
   }
   
   validates :first_name, presence: true
   validates :last_name, presence: true
-  validates :email, presence: true
   validates :gender, presence: true
-  # validates :birth_date, presence: true, format: { with: URI::MailTo::EMAIL_REGEXP }
+  validates :email, presence: true, format: { with: VALID_EMAIL_REGEX }
```

Comment:
Cool. I thought I tried that pattern and had the same trouble as with the library. Thanks for working this out.